Dockerfile.windows

```diff
@@ -102,11 +102,15 @@ RUN cmake --build . --config Release; `
 # TODO: Do something with /work/out/bin/fluent-bit.{exe,dll}
 
 ###############################################################################
-# Build OT collector
+# Build Go code in one container to exploit Go build caching
 ###############################################################################
 
 FROM base as gobuilder
 
+###############################################################################
+# Build OT collector
+###############################################################################
+
 COPY . /work
```

Comment:
This one is shared between the OT collector and the Ops agent, so I'd put this above the comment. Feel free to submit once done.

Reply:
I fixed it the other direction by adding a separate COPY for each clause. PTAL.